tests/out/wgsl/bevy-pbr-frag.wgsl

```diff
@@ -23,8 +23,8 @@ struct CameraPosition {
 struct Lights {
     AmbientColor: vec4<f32>;
     NumLights: vec4<u32>;
-    PointLights: [[stride(48)]] array<PointLight,10>;
-    DirectionalLights: [[stride(32)]] array<DirectionalLight,1>;
+    PointLights: [[stride(48)]] array<PointLight,10u>;
```

Comment:
this change shouldn't be needed. WGSL allows signed/unsigned. The frontend should generate only unsigned.

Reply:
That's what is happening here, the frontend always generates uints and the wgsl backend picks up the uints and writes them with the correct suffix, there's nothing that can be done here to prevent this (besides changing the backend)

Reply:
ah sorry, dummy me, you are right